kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -388,336 +356,281 @@
-        KAFKASTORE_SSL_PROVIDER_DOC
-    )
-    .define(
-        KAFKACLIENT_SSL_CIPHER_SUITES_CONFIG,
-        Type.STRING,
-        "",
-        Importance.LOW,
-        KAFKACLIENT_SSL_CIPHER_SUITES_DOC
-    )
-    .define(
-        KAFKACLIENT_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG,
-        Type.STRING,
-        "",
-        Importance.LOW,
-        KAFKACLIENT_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_KERBEROS_SERVICE_NAME_CONFIG,
-        Type.STRING,
-        "",
-        Importance.MEDIUM,
-        KAFKACLIENT_SASL_KERBEROS_SERVICE_NAME_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_MECHANISM_CONFIG,
-        Type.STRING,
-        "GSSAPI",
-        Importance.MEDIUM,
-        KAFKACLIENT_SASL_MECHANISM_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_KERBEROS_KINIT_CMD_CONFIG,
-        Type.STRING,
-        "/usr/bin/kinit",
-        Importance.LOW,
-        KAFKACLIENT_SASL_KERBEROS_KINIT_CMD_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_KERBEROS_MIN_TIME_BEFORE_RELOGIN_CONFIG,
-        Type.LONG,
-        60000,
-        Importance.LOW,
-        KAFKACLIENT_SASL_KERBEROS_MIN_TIME_BEFORE_RELOGIN_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_JITTER_CONFIG,
-        Type.DOUBLE,
-        0.05,
-        Importance.LOW,
-        KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_JITTER_DOC
-    )
-    .define(
-        KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR_CONFIG,
-        Type.DOUBLE,
-        0.8,
-        Importance.LOW,
-        KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR_DOC
-    )
-    .define(
-        KAFKA_REST_RESOURCE_EXTENSION_CONFIG,
-        Type.LIST,
-        "",
-        Importance.LOW,
-        KAFKA_REST_RESOURCE_EXTENSION_DOC
-    )
-    .define(
-        CRN_AUTHORITY_CONFIG,
-        Type.STRING,
-        CONFLUENT_RESOURCE_NAME_AUTHORITY_DEFAULT,
-        Importance.LOW,
-        CONFLUENT_RESOURCE_NAME_AUTHORITY_DOC
-    )
-    .define(
-        API_ENDPOINTS_ALLOWLIST_CONFIG,
-        Type.LIST,
-        API_ENDPOINTS_ALLOWLIST_DEFAULT,
-        Importance.LOW,
-        API_ENDPOINTS_ALLOWLIST_DOC
-    )
-    .define(
-        API_ENDPOINTS_BLOCKLIST_CONFIG,
-        Type.LIST,
-        API_ENDPOINTS_BLOCKLIST_DEFAULT,
-        Importance.LOW,
-        API_ENDPOINTS_BLOCKLIST_DOC
-    )
-    .define(
-        API_V2_ENABLE_CONFIG,
-        Type.BOOLEAN,
-        API_V2_ENABLE_DEFAULT,
-        Importance.LOW,
-        API_V2_ENABLE_DOC
-    )
-    .define(
-        API_V3_ENABLE_CONFIG,
-        Type.BOOLEAN,
-        API_V3_ENABLE_DEFAULT,
-        Importance.LOW,
-        API_V3_ENABLE_DOC);
+            KAFKAREST_PORT_DEFAULT,
+            KAFKAREST_LISTENERS_DEFAULT,
+            String.join(",", Versions.PREFERRED_RESPONSE_TYPES),
+            MediaType.APPLICATION_JSON,
+            METRICS_JMX_PREFIX_DEFAULT_OVERRIDE)
+        .define(ID_CONFIG, Type.STRING, ID_DEFAULT, Importance.HIGH, ID_CONFIG_DOC)
```

Comment:
Can the formatting be overriden here so all the defines match?

Reply:
Google Java Format is opinionated in the fact that you can't override or suppress anything.